src/numerics.jl

```diff
@@ -61,6 +63,43 @@ function compute_equilibrium!(discretization; max_years=100, rel_error=2e-5, ver
     return average_temperature
 end
 
+"""
+    compute_evolution!!(...)
+
+* rel_error is the tolerance for global temperature equilibrium (default is 2e-5).
+* max_years is the maximum number of annual cycles to be computed when searching for equilibrium
+"""
+function compute_evolution!(discretization, co2_concentration_yearly, mean_temperature_yearly, year_start; verbose=true)
```

Comment:
I would add `mean_temperature_yearly` as an output quantity, and would also output the mean temperature at each step

Reply:
makes sense!